### PR TITLE
Card type error

### LINKED ISSUE #1732: Improve messaging for template errors
This comes up fairly frequently:

https://forums.ankiweb.net/t/error-with-exporting/18483

Perhaps a few tweaks would help:

- showing just the relevant error, instead of a backtrace
- providing the user with a 'more info' button or link that explains how to deal with the problem

### PR DESCRIPTION
**Pull request by RumovZ**
Closes #1732.

### CODE REVIEW

File: pylib/anki/errors.py [hunk 1]
```diff
@@ -17,6 +18,14 @@ def __str__(self) -> str:
         return self._localized
 
 
+class DocumentedError(LocalizedError):
+    """A localized error described in the manual."""
+
+    def __init__(self, localized: str, help_page: Any) -> None:
```
Comment (RumovZ) [inline pylib/anki/errors.py:24]: I didn't get the type annotation to work with `anki.collection.HelpPage.V` or something similar.

File: qt/aqt/exporting.py [hunk 2]
```diff
@@ -174,10 +175,14 @@ def on_done(future: Future) -> None:
                 try:
                     # raises if exporter failed
                     future.result()
+                except CardTypeError as e:
```
Comment (RumovZ) [inline qt/aqt/exporting.py:178]: Or maybe `DocumentedError`? Not sure about its implementation in general.

**Comment by dae** [comment]
WDYT?

**Comment by RumovZ** [comment]
Thanks!

**Comment by dae** [comment]
Whoops - likewise!

**Pull request merged**